### ISSUE TITLE
Undeclared dependency Test::Exception

### ISSUE THREAD

**Issue by eserte** [khrt/Raisin#66]
t/unit/api.t fails if Test::Exception is not installed:
```
Can't locate Test/Exception.pm in @INC (you may need to install the Test::Exception module) (@INC contains: /home/cpansand/.cpan/build/2019062010/Raisin-0.83-2/blib/lib /home/cpansand/.cpan/build/2019062010/Raisin-0.83-2/blib/arch /home/cpansand/.cpan/build/2019062010/Type-Tiny-1.004004-1/blib/arch /home/cpansand/.cpan/build/2019062010/Type-Tiny-1.004004-1/blib/lib /home/cpansand/.cpan/build/2019062010/Exporter-Tiny-1.002001-0/blib/arch /home/cpansand/.cpan/build/2019062010/Exporter-Tiny-1.002001-0/blib/lib /home/cpansand/.cpan/build/2019062010/Plack-Middleware-CrossOrigin-0.014-1/blib/arch /home/cpansand/.cpan/build/2019062010/Plack-Middleware-CrossOrigin-0.014-1/blib/lib /home/cpansand/.cpan/build/2019062010/Plack-1.0047-1/blib/arch /home/cpansand/.cpan/build/2019062010/Plack-1.0047-1/blib/lib /home/cpansand/.cpan/build/2019062010/Test-TCP-2.19-1/blib/arch /home/cpansand/.cpan/build/2019062010/Test-TCP-2.19-1/blib/lib /home/cpansand/.cpan/build/2019062010/HTTP-Headers-Fast-0.22-1/blib/arch /home/cpansand/.cpan/build/2019062010/HTTP-Headers-Fast-0.22-1/blib/lib /home/cpansand/.cpan/build/2019062010/HTTP-Entity-Parser-0.21-1/blib/arch /home/cpansand/.cpan/build/2019062010/HTTP-Entity-Parser-0.21-1/blib/lib /home/cpansand/.cpan/build/2019062010/WWW-Form-UrlEncoded-0.26-1/blib/arch /home/cpansand/.cpan/build/2019062010/WWW-Form-UrlEncoded-0.26-1/blib/lib /home/cpansand/.cpan/build/2019062010/Stream-Buffered-0.03-1/blib/arch /home/cpansand/.cpan/build/2019062010/Stream-Buffered-0.03-1/blib/lib /home/cpansand/.cpan/build/2019062010/JSON-MaybeXS-1.004000-1/blib/arch /home/cpansand/.cpan/build/2019062010/JSON-MaybeXS-1.004000-1/blib/lib /home/cpansand/.cpan/build/2019062010/Cpanel-JSON-XS-4.12-1/blib/arch /home/cpansand/.cpan/build/2019062010/Cpanel-JSON-XS-4.12-1/blib/lib /home/cpansand/.cpan/build/2019062010/Hash-MultiValue-0.16-1/blib/arch /home/cpansand/.cpan/build/2019062010/Hash-MultiValue-0.16-1/blib/lib /home/cpansand/.cpan/build/2019062010/HTTP-MultiPartParser-0.02-1/blib/arch /home/cpansand/.cpan/build/2019062010/HTTP-MultiPartParser-0.02-1/blib/lib /home/cpansand/.cpan/build/2019062010/Filesys-Notify-Simple-0.13-1/blib/arch /home/cpansand/.cpan/build/2019062010/Filesys-Notify-Simple-0.13-1/blib/lib /home/cpansand/.cpan/build/2019062010/Test-SharedFork-0.35-1/blib/arch /home/cpansand/.cpan/build/2019062010/Test-SharedFork-0.35-1/blib/lib /home/cpansand/.cpan/build/2019062010/File-ShareDir-1.116-1/blib/arch /home/cpansand/.cpan/build/2019062010/File-ShareDir-1.116-1/blib/lib /home/cpansand/.cpan/build/2019062010/Class-Inspector-1.34-1/blib/arch /home/cpansand/.cpan/build/2019062010/Class-Inspector-1.34-1/blib/lib /home/cpansand/.cpan/build/2019062010/Devel-StackTrace-AsHTML-0.15-1/blib/arch /home/cpansand/.cpan/build/2019062010/Devel-StackTrace-AsHTML-0.15-1/blib/lib /home/cpansand/.cpan/build/2019062010/Devel-StackTrace-2.04-1/blib/arch /home/cpansand/.cpan/build/2019062010/Devel-StackTrace-2.04-1/blib/lib /home/cpansand/.cpan/build/2019062010/Cookie-Baker-0.11-1/blib/arch /home/cpansand/.cpan/build/2019062010/Cookie-Baker-0.11-1/blib/lib /home/cpansand/.cpan/build/2019062010/Test-Time-0.08-1/blib/arch /home/cpansand/.cpan/build/2019062010/Test-Time-0.08-1/blib/lib /home/cpansand/.cpan/build/2019062010/Apache-LogFormat-Compiler-0.35-1/blib/arch /home/cpansand/.cpan/build/2019062010/Apache-LogFormat-Compiler-0.35-1/blib/lib /home/cpansand/.cpan/build/2019062010/Test-Requires-0.10-1/blib/arch /home/cpansand/.cpan/build/2019062010/Test-Requires-0.10-1/blib/lib /home/cpansand/.cpan/build/2019062010/Test-MockTime-0.17-1/blib/arch /home/cpansand/.cpan/build/2019062010/Test-MockTime-0.17-1/blib/lib /home/cpansand/.cpan/build/2019062010/POSIX-strftime-Compiler-0.42-1/blib/arch /home/cpansand/.cpan/build/2019062010/POSIX-strftime-Compiler-0.42-1/blib/lib /home/cpansand/.cpan/build/2019062010/File-ShareDir-Install-0.13-1/blib/arch /home/cpansand/.cpan/build/2019062010/File-ShareDir-Install-0.13-1/blib/lib /home/cpansand/.cpan/build/2019062010/JSON-4.02-1/blib/arch /home/cpansand/.cpan/build/2019062010/JSON-4.02-1/blib/lib /home/cpansand/.cpan/build/2019062010/Type-Tiny-1.004004-1/blib/arch /home/cpansand/.cpan/build/2019062010/Type-Tiny-1.004004-1/blib/lib /home/cpansand/.cpan/build/2019062010/Exporter-Tiny-1.002001-0/blib/arch /home/cpansand/.cpan/build/2019062010/Exporter-Tiny-1.002001-0/blib/lib /home/cpansand/.cpan/build/2019062010/Plack-Middleware-CrossOrigin-0.014-1/blib/arch /home/cpansand/.cpan/build/2019062010/Plack-Middleware-CrossOrigin-0.014-1/blib/lib /home/cpansand/.cpan/build/2019062010/Plack-1.0047-1/blib/arch /home/cpansand/.cpan/build/2019062010/Plack-1.0047-1/blib/lib /home/cpansand/.cpan/build/2019062010/Test-TCP-2.19-1/blib/arch /home/cpansand/.cpan/build/2019062010/Test-TCP-2.19-1/blib/lib /home/cpansand/.cpan/build/2019062010/HTTP-Headers-Fast-0.22-1/blib/arch /home/cpansand/.cpan/build/2019062010/HTTP-Headers-Fast-0.22-1/blib/lib /home/cpansand/.cpan/build/2019062010/HTTP-Entity-Parser-0.21-1/blib/arch /home/cpansand/.cpan/build/2019062010/HTTP-Entity-Parser-0.21-1/blib/lib /home/cpansand/.cpan/build/2019062010/WWW-Form-UrlEncoded-0.26-1/blib/arch /home/cpansand/.cpan/build/2019062010/WWW-Form-UrlEncoded-0.26-1/blib/lib /home/cpansand/.cpan/build/2019062010/Stream-Buffered-0.03-1/blib/arch /home/cpansand/.cpan/build/2019062010/Stream-Buffered-0.03-1/blib/lib /home/cpansand/.cpan/build/2019062010/JSON-MaybeXS-1.004000-1/blib/arch /home/cpansand/.cpan/build/2019062010/JSON-MaybeXS-1.004000-1/blib/lib /home/cpansand/.cpan/build/2019062010/Cpanel-JSON-XS-4.12-1/blib/arch /home/cpansand/.cpan/build/2019062010/Cpanel-JSON-XS-4.12-1/blib/lib /home/cpansand/.cpan/build/2019062010/Hash-MultiValue-0.16-1/blib/arch /home/cpansand/.cpan/build/2019062010/Hash-MultiValue-0.16-1/blib/lib /home/cpansand/.cpan/build/2019062010/HTTP-MultiPartParser-0.02-1/blib/arch /home/cpansand/.cpan/build/2019062010/HTTP-MultiPartParser-0.02-1/blib/lib /home/cpansand/.cpan/build/2019062010/Filesys-Notify-Simple-0.13-1/blib/arch /home/cpansand/.cpan/build/2019062010/Filesys-Notify-Simple-0.13-1/blib/lib /home/cpansand/.cpan/build/2019062010/Test-SharedFork-0.35-1/blib/arch /home/cpansand/.cpan/build/2019062010/Test-SharedFork-0.35-1/blib/lib /home/cpansand/.cpan/build/2019062010/File-ShareDir-1.116-1/blib/arch /home/cpansand/.cpan/build/2019062010/File-ShareDir-1.116-1/blib/lib /home/cpansand/.cpan/build/2019062010/Class-Inspector-1.34-1/blib/arch /home/cpansand/.cpan/build/2019062010/Class-Inspector-1.34-1/blib/lib /home/cpansand/.cpan/build/2019062010/Devel-StackTrace-AsHTML-0.15-1/blib/arch /home/cpansand/.cpan/build/2019062010/Devel-StackTrace-AsHTML-0.15-1/blib/lib /home/cpansand/.cpan/build/2019062010/Devel-StackTrace-2.04-1/blib/arch /home/cpansand/.cpan/build/2019062010/Devel-StackTrace-2.04-1/blib/lib /home/cpansand/.cpan/build/2019062010/Cookie-Baker-0.11-1/blib/arch /home/cpansand/.cpan/build/2019062010/Cookie-Baker-0.11-1/blib/lib /home/cpansand/.cpan/build/2019062010/Test-Time-0.08-1/blib/arch /home/cpansand/.cpan/build/2019062010/Test-Time-0.08-1/blib/lib /home/cpansand/.cpan/build/2019062010/Apache-LogFormat-Compiler-0.35-1/blib/arch /home/cpansand/.cpan/build/2019062010/Apache-LogFormat-Compiler-0.35-1/blib/lib /home/cpansand/.cpan/build/2019062010/Test-Requires-0.10-1/blib/arch /home/cpansand/.cpan/build/2019062010/Test-Requires-0.10-1/blib/lib /home/cpansand/.cpan/build/2019062010/Test-MockTime-0.17-1/blib/arch /home/cpansand/.cpan/build/2019062010/Test-MockTime-0.17-1/blib/lib /home/cpansand/.cpan/build/2019062010/POSIX-strftime-Compiler-0.42-1/blib/arch /home/cpansand/.cpan/build/2019062010/POSIX-strftime-Compiler-0.42-1/blib/lib /home/cpansand/.cpan/build/2019062010/File-ShareDir-Install-0.13-1/blib/arch /home/cpansand/.cpan/build/2019062010/File-ShareDir-Install-0.13-1/blib/lib /home/cpansand/.cpan/build/2019062010/JSON-4.02-1/blib/arch /home/cpansand/.cpan/build/2019062010/JSON-4.02-1/blib/lib /opt/perl-5.28.2t/lib/site_perl/5.28.2/x86_64-linux-thread-multi /opt/perl-5.28.2t/lib/site_perl/5.28.2 /opt/perl-5.28.2t/lib/5.28.2/x86_64-linux-thread-multi /opt/perl-5.28.2t/lib/5.28.2 .) at t/unit/api.t line 6.
BEGIN failed--compilation aborted at t/unit/api.t line 6.
t/unit/api.t .......................... 
Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
```

**Comment by eserte** [comment]
Maybe also undeclared: JSON::MaybeXS:
```
    #   Failed test at t/behaviour/app-routes.t line 76.
    #          got: '500'
    #     expected: '200'
    # Can't locate JSON/MaybeXS.pm in @INC (you may need to install the JSON::MaybeXS module) (@INC contains: ...  .) at /home/cpansand/.cpan/build/2019062101/Raisin-0.83-KD4DDl/blib/lib/Raisin/Encoder/JSON.pm line 6.
    # BEGIN failed--compilation aborted at /home/cpansand/.cpan/build/2019062101/Raisin-0.83-KD4DDl/blib/lib/Raisin/Encoder/JSON.pm line 6.
    # Compilation failed in require at /usr/share/perl5/Plack/Util.pm line 23.
Bailout called.  Further testing stopped:  FAILED!
FAILED--Further testing stopped: FAILED!
```

**Comment by khrt** [comment]
Fixed by #67
